framework/wazuh/rbac/orm.py

```diff
@@ -2030,6 +2144,46 @@ def replace_user_role(self, user_id: int, actual_role_id: int, new_role_id: int,
 
         return False
 
+    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
+                     to_id: Optional[int] = None):
```

Comment:
Please add the docstring

```suggestion
    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
                     to_id: Optional[int] = None) -> None:
```

Reply:
Fixed in 402c4224dd39f3a6dd3cefca2e48f78cd7c872a2.